python/ray/dashboard/dashboard.py

```diff
@@ -385,7 +385,7 @@ def __init__(self, redis_address, redis_password=None):
             "children": {},
             "currentTaskFuncDesc": [],
             "ipAddress": "",
-            "isDirectCall": False,
+            "isDirectCall": True,
```

Comment:
Should we just remove this field?

Reply:
Was causing issues earlier but I'll try to track it down.